src/pages/CreateAccount.js

```diff
@@ -83,107 +64,90 @@ export default function CreateAccount({ api, handleSuccessfulSignIn }) {
+              name={"invite-code"}
+              placeholder={strings.inviteCodePlaceholder}
               value={inviteCode}
-              onChange={(e) => setInviteCode(e.target.value)}
-              placeholder={strings.code}
+              onChange={handleInviteCodeChange}
             />
-          </div>
 
-          <div className="inputField">
-            <label>{strings.name}</label>
-            <input
-              placeholder={strings.name}
+            <InputField
+              type={"text"}
+              label={strings.fullName}
+              id={"name"}
+              name={"name"}
+              placeholder={strings.fullNamePlaceholder}
               value={name}
-              onChange={(e) => setName(e.target.value)}
+              onChange={handleNameChange}
             />
-          </div>
 
-          <div className="inputField">
-            <label>{strings.email}</label>
-            <input
-              placeholder={strings.email}
+            <InputField
+              type={"email"}
+              label={strings.email}
+              id={"email"}
+              name={"email"}
+              placeholder={strings.emailPlaceholder}
               value={email}
-              onChange={(e) => setEmail(e.target.value)}
+              onChange={handleEmailChange}
             />
-          </div>
 
-          <div className="inputField">
-            <label>{strings.password}</label>
-            <input
-              type="password"
-              placeholder={strings.password}
+            <InputField
+              type={"password"}
+              label={strings.password}
+              id={"password"}
+              name={"password"}
+              placeholder={strings.passwordPlaceholder}
               value={password}
-              onChange={(e) => setPassword(e.target.value)}
-            />
-          </div>
-
-          <div className="inputField">
-            <label>{strings.learnedLanguage}</label>
-
-            <Select
-              elements={systemLanguages.learnable_languages}
-              label={(e) => e.name}
-              val={(e) => e.code}
-              updateFunction={setLearned_language}
+              onChange={handlePasswordChange}
+              helperText={strings.passwordHelperText}
             />
-          </div>
-
-          <div className="inputField">
-            <label>{strings.levelOfLearnedLanguage}</label>
-
-            <Select
-              elements={CEFR_LEVELS}
-              label={(e) => e.label}
-              val={(e) => e.value}
-              updateFunction={setLearned_cefr_level}
+          </FormSection>
+          <FormSection>
+            <Checkbox
+              label={
+                <>
+                  By checking this box you agree to our{" "}
+                  <a
+                    className="bold underlined-link"
+                    href="https://raw.githubusercontent.com/zeeguu/browser-extension/main/PRIVACY.md"
```

Comment:
Should we open a new tab? Is replacing the current page the intended behaviour?

Reply:
We could modify it for the desktop to open a new tab, but for mobile, I would recommend to keep replacing the tab because navigating between tabs is not user-friendly on smartphones (especially for less tech-savvy users).

Reply:
I see! That reason sounds good to me, I was just double checking 👍

Reply:
I have just made a minor update and now the Privacy Policy opens in the same tab on mobile and in a separate one on desktop. We can also discuss possible ways of data preservation tomorrow 🙂